### PR TITLE
Fix: fixed the issues on topbar

### LINKED ISSUE #269: [BUG] items are displaying over the top bar when hovering
### Is there any existing issue for this?

- [X] I have searched the existing issues

### Description

when hovering over a subsection item while it on the top bar 
it displays over the top bar and makes it kinda look weird

### Screenshots

image

### Any additional information?

i can take on  this

### What browser are you seeing the problem on?

Chrome

### PR DESCRIPTION
**Pull request by dev-phantom**
## Fixes Issue
Closes #269 

## Changes proposed
removed general margin left from the general layout and added an individual padding to the topbar and cards

## Screenshots
image

### CODE REVIEW

**Comment by JunDevHarsh** [comment]
Hello @dev-phantom, I got the following issues while dealing with your changes made to the project.

LinksHub website preview: 

https://user-images.githubusercontent.com/94737463/221757699-1b5a4b63-ee85-4b9a-b804-6a3d06fc3e15.mp4

---

Your deployed preview:

https://user-images.githubusercontent.com/94737463/221757402-e099c1d9-36d5-4a14-9a76-a682f2d0e92d.mp4

**Comment by dev-phantom** [comment]
> Hello @dev-phantom, I got the following issues while dealing with your changes made to the project.
> 
> LinksHub website preview: 
> 
> https://user-images.githubusercontent.com/94737463/221757699-1b5a4b63-ee85-4b9a-b804-6a3d06fc3e15.mp4
> 
> ---
> 
> Your deployed preview:
> 
> https://user-images.githubusercontent.com/94737463/221757402-e099c1d9-36d5-4a14-9a76-a682f2d0e92d.mp4
> 
> 

Thanks for spotting that out I'll look into it

File: components/TopBar/TopBar.tsx [hunk 1]
```diff
@@ -7,10 +7,10 @@ export const TopBar = ({ header }: { header?: string }) => {
   const router = useRouter();
   const category = router.asPath.replace("/", "");
   return (
-    <div className="sticky top-0 dark:bg-[#101623] bg-gray-100 transition-all ease-in duration-300 w-full xs:overflow-x-hidden mb-4 flex flex-col-reverse items-center md:flex-row md:justify-between md:py-4">
-      <div className="my-2 flex items-center place-self-start text-lg dark:text-gray-300 sm:text-3xl">
+    <div className="sticky z-[999] top-0 dark:bg-[#101623] bg-gray-100 transition-all ease-in duration-300 w-full xs:overflow-x-hidden mb-4 flex flex-col-reverse items-center md:flex-row md:justify-between md:py-4">
```
Comment (JunDevHarsh) [inline components/TopBar/TopBar.tsx:10]: Giving the `z-index` value to `1` would solve this issue.

**Comment by dev-phantom** [comment]
@JunDevHarsh please recheck if the issue still persist.

**Review by JunDevHarsh — APPROVED** [review]

**Review by rupali-codes — APPROVED** [review]
lgtm thanks :)

**Pull request merged**